### ISSUE TITLE
Random swap-source image from a folder, and or a single photo

### ISSUE THREAD

**Issue by ahdsr** [Gourieff/Roop-GE#2]
It would be great to pull a random photo from a folder, when it comes to automating prompts and such

**Comment by Gourieff** [comment]
Do you mean any random photo in a specific folder?

**Comment by ahdsr** [comment]
Yeah, random or in order, as long as you can specify a directory, or it can be tied to the batch # amount.
